### DISCUSSION TITLE
add search feature when browsing code

### DISCUSSION THREAD

**Discussion by jdhao** [community/community#6578] [Mobile] [answered]
When browsing the code, there is no option to search the source file.

**Comment by aconsuegra** [comment, marked as answer]
Thank you so much for the request! We are tracking being able to add search capabilities to our code view internally and I added your note to our issue. In the meantime, please let us know if you have any other bugs or feature requests. We really appreciate it!

**Comment by BrandowLucas** [comment]
+1
